client/app/models/contact.coffee

```diff
@@ -148,6 +148,26 @@ module.exports = class Contact extends Backbone.Model
         _.compact([pf, fn, mn, gn, sf]).join ' '
 
 
+    # We override toJSON to ensure the exported object is the a true object.
+    # Allow a options.toVCF boolean to sanitize the output before.
+    toJSON: ({toVCF} = {}) ->
+        data = super
+        data.datapoints = data.datapoints.toJSON()
+
+        return data unless toVCF
+
+        # Isolate in a try..catch block to go outside main thread and pop error
+        try
+            # We remove the BDay datapoint if it doesn't match ISO 8601
+            unless moment(data.bday, 'YYYY-MM-DD', true).isValid()
+                delete data['bday']
+                throw new Error "Malformed bday for #{data.id}"
+        catch err
+            console.error err
```

Comment:
Definitely agree! Wht's I have in mind is probably use a notification bus system to send the error in, and let the platform do the magic. At this time, we doesn't have it, nor a local notification system into the app (we never have the use it since now).

So, I just choose to fallback to a basic `console.error`for the moment, and let's see when a notification system'll be available :).